BSB_lan_defs.h

```diff
@@ -7678,9 +7773,12 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 {0x593D1769,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8430,  STR8430,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Quelle Austritt Min
 {0x053D195A,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8431,  STR8431,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Quellenzw'kreis Vorlauftemp
 {0x053D196D,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8434,  STR8434,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Sauggastemperatur
-{0x053D163B,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8435,  STR8435,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Verdampfungstemperatur
-{0x593D1662,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8436,  STR8436,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Überhitzung oder Überhitzungssollwert
-// vorheriges könnte auch 1663 sein weil da noch Überhitzungssollwert übermittelt wird
+{0x593D1664,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8435,  STR8435,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Verdampfungstemperatur
+// todo: How to integrate this line? it's sent along with parameter 8435
+// {0x053D163B,  CAT_DIAG_ERZEUGER,    VT_PRESSURE_WORD, ????,  STR8435_1, 0,                   NULL,         FL_RONLY,     DEV_ALL}, // Verdampfungsdruck
```

Comment:
Not sure which parameter number to assign. The datagram is sent and displayed along with parameter 8435.

Reply:
Same as above, we have a report from the same user (device family 108) that 0x053D163B is "Verdampfungstemperatur". If you proceed as above, 8436 should be kept, but modified from DEV_ALL to DEV_108 and your new line should then be DEV_ALL.

Reply:
Ok, forget what I wrote about 8436 because you have a new parameter below which I just saw...